src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -3,6 +3,25 @@
 {% set title = _('Status & Versions') %}
 {% block title %}{{ dev_page_title(title, addon) }}{% endblock %}
 
+{% if addon.has_listed_versions() %}
+  {% set has_listed_versions = True %}
+{% endif %}
+{% if addon.has_unlisted_versions() %}
+  {% set has_unlisted_versions = True %}
+{% endif %}
+
+{% if has_listed_versions and has_unlisted_versions %}
+  {% macro distro_tag(channel) %}
```

Comment:
Do we really need to make the label depend on whether or not the addon has mixed versions ? I'd have just made it depend on the `mixed-listed-unlisted` waffle and always display it when it's enabled.